### PR TITLE
Csip/dmd sec/csip21

### PR DESCRIPTION
**Pull request by PhillipAasvangTommerholt**

### CODE REVIEW

File: corpora/csip/metadata/dmdsec/CSIP21/testCase.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!-- Root element for an individual test case, allows these to be wrapped into
 XML lists -->
 <testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd"
-  testable="UNKNOWN">
+  testable="TRUE">
   <!-- Unique ID for the test case -->
   <id specification="E-ARK CSIP" version="2.0-DRAFT" requirementId="CSIP21"/>
```
Comment (scfkmd) [inline corpora/csip/metadata/dmdsec/CSIP21/testCase.xml]: Change to 2.0.4

File: corpora/csip/metadata/dmdsec/CSIP21/testCase.xml [hunk 2]
```diff
@@ -10,14 +10,40 @@ XML lists -->
...
+    The problem here is that the correctness of there being a mdRef-element or not really relies on whether there is a descriptive metadata file or not. The current CSIP21 does not take this into consideration.</description>
   <!-- List of requirments that this test case depends on in addition to the
   main requirememt, e.g. general requirments on the form of IDs -->
   <dependencies>
   </dependencies>
   <!-- A list of the validation rules derived from the test case -->
   <rules>
+    <rule id="1">
+      <description>The mdRef-element SHOULD be there</description>
+      <error level="ERROR">
```
Comment (scfkmd) [inline corpora/csip/metadata/dmdsec/CSIP21/testCase.xml]: Change to warning

File: corpora/csip/metadata/dmdsec/CSIP21/testCase.xml [hunk 3]
```diff
@@ -10,14 +10,40 @@ XML lists -->
...
+      </error>
+      <corpusPackages>
+        <package name="IP_18000_CSIP21_1" isValid="TRUE" >
+          <path>/corpora/csip/metadata/dmdSec/CSIP21/IP_18000_CSIP21_1</path>
+          <description>A package based on IP_18006_SIARD2_1Rep_externallobs. Contains a dmdSec element with no mdRef-element in both root METS.xml and representation METS but with files in both.</description>
+        </package>
+      </corpusPackages>
+    </rule>
+    <rule id="2">
+      <description>The mdRef-element SHOULD be there</description>
```
Comment (scfkmd) [inline corpora/csip/metadata/dmdsec/CSIP21/testCase.xml:37]: Change to "MUST" be there.

**Review by scfkmd — CHANGES_REQUESTED** [review]
Partially done with review

File: corpora/csip/metadata/dmdsec/CSIP21/testCase.xml (hunk 3 above)
Comment (PhillipAasvangTommerholt) [reply corpora/csip/metadata/dmdsec/CSIP21/testCase.xml:37]: done

**Review by scfkmd — APPROVED** [review]
Changes approved

**Review by scfkmd — APPROVED** [review]
Approved.

**Review by ghost — APPROVED** [review]
Changes approved.

**Pull request merged**